PKHeX.Core/Legality/Checks.cs

```diff
@@ -419,7 +419,7 @@ private void VerifyTradeOTOnly(string[] validOT)
             if (pkm.IsNicknamed)
                 AddLine(Severity.Invalid, V9, CheckIdentifier.Nickname);
             int lang = pkm.Language;
-            if (validOT.Length >= lang)
```

Comment:
This looks like a guard for `validOT[lang]`. Wouldn't this cause issues if lang is actually greater than validOT.Length?

Reply:
it's more obvious if you flip the two arguments around
it's currently flagging if lang is less than, should be flagging if lang >= validOT.Length